src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -89,7 +89,13 @@ public bool IsAnalysisSupported(IEnumerable<AnalysisLanguage> languages)
             TriggerAnalysisAsync(request, consumer, cancellationToken)
                 .Forget(); // fire and forget
 
-        private async Task TriggerAnalysisAsync(Request request, IIssueConsumer consumer, CancellationToken cancellationToken)
+        protected /* for testing */ virtual void CallSubProcess(Action<Message> handleMessage, Request request, IAnalysisStatusNotifier analysisStatusNotifier, ISonarLintSettings settings,
```

Comment:
I originally define a delegate, but it looked weird to have a mix of delegates and `virtual override` points in the class (the existing two virtual methods are defined immediately above this one).
I also looked at creating an interface, but that involved more change. This was the minimum change I could make to the product code to make the behaviour testable.